src/style.css

```diff
@@ -0,0 +1,120 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+  align-self: center;
+}
+
+.card__content {
+  padding-top: 40px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  padding-top: 4px;
+  color: #616070;
+}
+
+.stars {
+  display: flex;
+  height: 16px;
+  gap: 4px;
+  width: 96px;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star :nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.block__stars {

```

Comment:
There should be only 2 BEM blocks in your code card and stars all the others are just card elements

Reply:
check it out everywhere

Reply:
I don't know how to add data-qa attribute...

    <link rel="stylesheet" href="./styles.css/styles-main.css">
  </head>
  <body>
    <div
      class="card"
      data-qa="card">
      <img 
        src="images/imac.jpeg"
        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
        class="card__image"
      >
      <h2 class="card__content">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h2>